tests/unit/testUtils.ts

```diff
@@ -59,3 +61,32 @@ export async function getTestStorageManager(): Promise<StorageManager> {
   await getTestExtensionContext();
   return StorageManager.getInstance();
 }
+
+/** Create a Kafka TopicData instance, as if from a REST response. */
+export function createTestTopicData(
+  clusterId: string,
+  topicName: string,
+  authorizedOperations: KafkaTopicOperation[],
+): TopicData {
+  return {
+    kind: "KafkaTopic",
+    metadata: {
+      self: "test",
+    },
+    cluster_id: clusterId,
+    topic_name: topicName,
+    is_internal: false,
+    replication_factor: 1,
+    partitions_count: 3,
+    partitions: {
+      related: "test",
+    },
+    partition_reassignments: {
+      related: "test",
+    },
+    configs: {
+      related: "test",
+    },
+    authorized_operations: authorizedOperations,
+  };
+}
```

Comment:
May not impact much currently, but to carry over one lesson from the recent `Connection` typing work: we may want to similarly use a `TopicDataFromJSON()` wrapper for these kinds of test util functions to ensure all properties are available in the expected formats

Reply:
Done.